src/SFA.DAS.Tools.Servicebus.Support.Infrastructure/Services/CosmosDb/CosmosMessageDbContext.cs

```diff
@@ -222,5 +223,24 @@ private string AddPaging(string sqlQuery, SearchProperties searchProperties)
         }
 
         private static string AddTypeClause(string sqlQuery) => sqlQuery + " AND c.type='message'";
+        
+        public async Task<IEnumerable<UserMessageCount>> GetMessageCountPerUser()
```

Comment:
Add async for consistency
```suggestion
        public async Task<IEnumerable<UserMessageCount>> GetMessageCountPerUserAsync()
```